src/main/webapp/app/exercises/shared/example-submission/example-submissions.component.ts

```diff
@@ -69,4 +83,28 @@ export class ExampleSubmissionsComponent implements OnInit {
             ];
         }
     }
+
+    /**
+     * Opens the import module for example submission
+     * Then invokes import api for selected submission
+     */
+    openImportModal() {
+        const exampleSubmissionImportModalRef = this.modalService.open(ExampleSubmissionImportComponent, {
+            size: 'lg',
+            backdrop: 'static',
+        });
+        exampleSubmissionImportModalRef.componentInstance.exercise = this.exercise;
+        exampleSubmissionImportModalRef.result.then(
+            (selectedSubmission: Submission) => {
+                this.exampleSubmissionService.import(selectedSubmission.id!, this.exercise.id!).subscribe(
+                    () => {
+                        this.alertService.success('artemisApp.exampleSubmission.submitSuccessful');
+                        location.reload();
+                    },
+                    (error: HttpErrorResponse) => onError(this.alertService, error),
+                );
+            },
+            () => {},
```

Comment:
I think,`()=>{}` is not necessary.

Reply:
additionally, I do not know how to make Client Test green 😢